libobs/obs-windows.c

```diff
@@ -107,6 +107,18 @@ static void log_processor_cores(void)
 	     os_get_physical_cores(), os_get_logical_cores());
 }
 
+static void log_emulation_status(void)
+{
+#if defined(_M_ARM64EC)
+	blog(LOG_INFO, "Windows ARM64: Compatible with x64 plugins");
```

Comment:
I don't understand this sentence. What is it communicating?

Reply:
Why was this marked as resolved? Nothing has been clarified regarding the logged string or the line of code.

Reply:
ARM64EC is a new ARM64 ABI in Windows that is compatible with the x64 emulator and can load x64 libraries.

In OBS, that would be a special OBS build which is native by itself but can some of its plugins still being x64.

Note ARM64 and ARM64EC are not inter-compatible but ARM64EC and x64 are.

I don't think ARM64 would be that useful in the context of OBS though.